calsub/renderical.go

```diff
@@ -52,27 +41,23 @@ END:VEVENT
 END:VCALENDAR
 `, "\n", "\r\n")))
 
-func (cs Subscription) renderICalFromShifts(appName string, shifts []oncall.Shift, generatedAt time.Time) ([]byte, error) {
-	var eventUIDs []string
-	for _, s := range shifts {
+func (r renderData) renderICal() ([]byte, error) {
```

Comment:
nit: missing method description (what is it, what's the high level purpose of the method?)